blocks/feed-newsroom/feed-newsroom.css

```diff
@@ -0,0 +1,123 @@
+/* eslint no-empty: "info" */
+
+.block.highlight.feed-newsroom > .others {
+    display: flex;
+    flex-direction: column;
+  }
+
+.block.highlight.feed-newsroom > .others picture {
+    width: auto;
+    height: 250px;
+  }
+
+main .block.feed-newsroom {
+    gap: 10vw;
+}
+
+.block.highlight.feed-newsroom > .others .title, .link-wrapper {
+    color: var(--link-color);
+}
+
+.block.highlight.feed-newsroom > .others h4 {
+    color: var(--text-color);
+}
+
+main .block.feed-newsroom > .others:first-child {
+    flex-grow: 3;
+}
+
+main .feed-newsroom-wrapper form {
+    margin: 50px;
+}
+
+main .feed-newsroom-wrapper form .filter-nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+main .feed-newsroom-wrapper button {
+    background-color: var(--secondary);
+    color: var(--white);
+    font-size: var(--body-font-size-m);;
+}
+
+
+main .feed-newsroom-wrapper form button {
+    min-width: 6rem;
+    background: var(--white);
+    color: var(--secondary);
+    padding: 6px 15px;
+    height: 32px;
+    font-weight: 500;
+    border-radius: 2px;
+    text-align: center;
+    font-size: var(--body-font-size-m);
+    border: 1px solid #007CAD;
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+}
+
+main .feed-newsroom-wrapper form button:hover {
+    background: #007cad;
```

Comment:
[Use this](https://github.com/hlxsites/sunstar-foundation/blob/30c60277a7599b4d55407d3cfbb48b1d4c7ca3e4/styles/styles.css#L38)